src/main/charts/confluence/templates/NOTES.txt

```diff
@@ -51,5 +51,11 @@ Get the {{ title .Chart.Name }} URL by running these commands in the same shell:
 ######            Data will be lost when the pod is terminated.             #####
 #################################################################################
 {{- end }}
+{{ if ( and .Values.synchrony.enabled ( not ( or .Values.volumes.synchronyHome.persistentVolumeClaim.create .Values.volumes.synchronyHome.customVolume ))) }}
+#################################################################################
+######      WARNING: Persistent volume is not used for Synchrony Home!!!    #####
+######            Data will be lost when the pod is terminated.             #####
```

Comment:
This looks very scary. I think we can say "Synchrony logs will be lost...". We are not losing Confluence or collaborative editing data.

Reply:
I agree with that. I've modified the warning message.